File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Extensibility/1033/ToolboxControlProject_CSharp/ToolboxControl.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace $safeprojectname$
{
    [ProvideToolboxControl("$projectname$", false)]
    public partial class ToolboxControl : UserControl
    {
        public ToolboxControl()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            MessageBox.Show(string.Format(System.Globalization.CultureInfo.CurrentUICulture, "$message$", this.ToString()));
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Extensibility/1033/ToolboxControlProject_VB/ToolboxControl.vb
Imports System.Windows.Forms

<ProvideToolboxControl("$projectname$", false)>
Public Class ToolboxControl

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        MessageBox.Show(String.Format(System.Globalization.CultureInfo.CurrentUICulture, "$message$", Me.ToString()))
    End Sub

End Class
